extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -558,10 +561,16 @@ export class AmpConsent extends AMP.BaseElement {
    * @return {Promise}
    */
   initPromptUI_(instanceId) {
-
     const promptUI = this.consentConfig_[instanceId]['promptUI'];
-    const element = this.getAmpDoc().getElementById(promptUI);
-    this.consentUI_[instanceId] = element;
+    if (promptUI) {
+      let element = this.getAmpDoc().getElementById(promptUI);
```

Comment:
can you do `this.element.getElementById(promtUI)` to save a `contains()` check.

Reply:
can't do that. no such function : )